### ISSUE TITLE
Missing Jammu and Kashmir (JK) AC's in Vidhan Sabha in India 

### ISSUE THREAD

**Issue by irenelfeng** [opencivicdata/ocd-division-ids#249]
I think there are missing assembly constituency ocd-ids for Jammu and Kashmir (i think there are 87). I am happy to make a PR and append them to the vidhan_sabha.csv - just confirming we need them.

**Comment by jloutsenhizer** [comment]
I believe the Vidhan Sabha constituencies were excluded in #223 due to an ongoing delimitation exercise. @novitam might be able to provide more details.

**Comment by irenelfeng** [comment]
ah, i see. Jammu and Kashmir's legislature was recently dissolved so boundaries are being redrawn. makes sense. When will these be added back? 

Another question I have surrounding India is what to do with cities within districts.  Would a proper ocd-id for Itanagar be, for example: 
`ocd-division/country:in/state:ar/district:pa/place:itanagar` 

since it is in the district of Papumpare? following what i see in: https://github.com/opencivicdata/ocd-division-ids/blob/master/identifiers/country-in/districts.csv

**Comment by novitam** [comment]
We don't know yet when the Jammu and Kashmir will be added back, we are waiting for the delimitation exercise to be completed. 

If you are referring to the Vidhan Sabha constituency of Itanagar in Papumpare, we have the following ocd-id:
`ocd-division/country:in/state:ar/district:pa/cd:itanagar` in https://github.com/opencivicdata/ocd-division-ids/blob/master/identifiers/country-in/vidhan_sabha.csv

We are using `cd:itanagar` because in this case we are referring to the electoral boundary, not the city boundary as described in https://github.com/opencivicdata/ocd-division-ids/blob/master/identifiers/country-in/README.

**Comment by irenelfeng** [comment]
Thank you @novitam! I can leave this issue open until they are imported or feel free to close. 

As for itanagar. I would actually like to describe the city boundary! Would using `place` be sufficient?  In the README this ocd-id type isn't specified but looks like city boundaries (for mayoral offices) aren't covered.

**Comment by novitam** [comment]
Yes, city boundaries for mayoral offices are not yet covered. I think you can create a PR to add a new file under `country-in` to cover the mayoral offices. I am not very familiar with the mayoral offices for India but `place` is very generic, so if there is a better term that is more specific, like `city` or maybe even `municipality` , whichever is more accurate for the boundary of mayoral offices, that would be better.
